spec.md

```diff
@@ -785,7 +785,7 @@ Enumerations are disjoint union types.
 
 Each enumeration consists of a set of variants.
 Each variant is named with a tag.
-Each variant also has a type associated with it which must be connectable (see [@sec:connectable-types]) and passive (see [@sec:passive-types]).
+Each variant also has a type associated with it which must be connectable (see [@sec:connectable-types]), storable (see [@sec:storable-types]), passive (see [@sec:passive-types]).
```

Comment:
I am not sure about this one. I'm not sure what the intended restrictions on enums are. Are we trying to keep Properties and Probes out?

Reply:
Well, I guess they are already excluded by the requirement on `connectable`. I am not sure why enums would care about being storable? Unless we would want to change the definition below to say say "an Enum is storable if all of its variants are storable"?

Reply:
Good point. I could relax this to say that enums are storable when the type of each variant is storable.

Reply:
Changed.